src/main/mal/DataResources.mal

```diff
@@ -88,96 +45,116 @@ category DataResources {
 
       | extract
         user info: "The attacker is able to extract the information. This means that they have a path available through which they can transfer the information back to a system they control."
+
+      // The following 10 attack steps are used to implement data replication
```

Comment:
Are there 10 attack steps after this line?

Reply:
There were before I removed 5 of them in a later commit. Funny catch. :)